owtf.py

```diff
@@ -54,7 +54,7 @@ def process_options(user_args):
 
     # Default settings:
     profiles = {}
-    plugin_group = arg.PluginGroup
+    plugin_group = arg.PluginGroup or 'web'
```

Comment:
If the command line option -g is passed the plugin_group will be set to the specified group, else `None` is set in `arg.PluginGroup`. In the case if no group is specified we set it to `web` to make `-t ` functional with `web` plugin_group. The default is set to `web` as discussed in issue #390 .